include/EVT/dev/LCD.hpp

```diff
@@ -0,0 +1,70 @@
+#ifndef _EVT_LCD_
+#define _EVT_LCD_
+
+#include <EVT/io/GPIO.hpp>
+#include <EVT/io/SPI.hpp>
+#include <EVT/io/UART.hpp>
+#include <EVT/io/pin.hpp>
+#include <EVT/utils/time.hpp>
+#include <stdint.h>
+
+#define ADCSELECT 0xA0
+#define DISPLAYOFF 0xAE
+#define COMDIRSCAN 0xC8
+#define LCDBIASET 0xA2
+#define POWERCONTROLSET 0x2F
+#define RESRATIOSET 0x26
+#define ELECTRONICVOLCOMMAND 0x81
+#define ELECTRONICVOLVALUE 0x11
+#define DISPLAYON 0xAF
+
+namespace EVT::core::DEV {
+/*
+* This class represents the structure to command a GLCD with 
+* a ST7565 controller.
+*/
+class LCD {
+
+public:
+    /// bit map to display the LCD's current state
+    unsigned char* bitMap;
+
+    ///
+    /*
+    * Constructor for the LCD class
+    *
+    * @param[in]
+    *
+    */
+    LCD(EVT::core::IO::GPIO& regSelect, EVT::core::IO::GPIO& reset, EVT::core::IO::SPI& spi, unsigned char* bitMap);
+
+    /// function to write data to the LCD
```

Comment:
Even though you're only writing one line of documentation for this, it should still be documented with /** **/ like the rest of the code. I'm picturing something like this:
```
/**
 * Write data to the LCD
 */
```